FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -16,6 +16,13 @@ protocol IssuesViewControllerDelegate : class {
 }
 
 class IssuesViewController : UITableViewController {
+
+    @IBInspectable var shouldFetchAllIssues: Bool = false {
+        didSet {
+            issuesManager.shouldFetchAllIssues = true
```

Comment:
I'm not a fan of setting a property and then later firing a method, and the method's behavior is dependent on this (especially since these two steps are separated physically in the code).

I'd like to move the logic into the `fetchIssues` method directly.  We can leverage the same enum for this.